tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/Capability/CapabilitySteps.cs

```diff
@@ -13,7 +13,7 @@ namespace NHSD.BuyingCatalogue.API.IntegrationTests.Steps.Capability
     [Binding]
     internal sealed class CapabilitySteps
     {
-        private const string ListCapabilitiesUrl = "http://localhost:8080/api/v1/Capabilities";
+        private const string ListCapabilitiesUrl = "http://localhost:5200/api/v1/Capabilities";
```

Comment:
The base address is hardcoded in quite a few places. Could we add an additional task to change this to use application settings?